TypeScript/src/components/Order.ts

```diff
@@ -0,0 +1,120 @@
+type MaybeNumber = number | null;
+
+export class Order{
+    bagsNumber:MaybeNumber;
+    private readonly coffePrice:MaybeNumber;
+    private readonly largeBoxPrice:MaybeNumber;
```

Comment:
let's try not to use MaybeNumber, we need to be extremely sure of what we are sending, and we should always check before using it that it isn't null.

Reply:
ok